app/code/core/Mage/Newsletter/etc/widget.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0"?>
+<!--
+/**
+ * OpenMage
+ *
+ * This source file is subject to the Academic Free License (AFL 3.0)
+ * that is bundled with this package in the file LICENSE_AFL.txt.
+ * It is also available at https://opensource.org/license/afl-3-0-php
+ *
+ * @category   Mage
+ * @package    Mage_Newsletter
+ * @copyright  Copyright (c) 2006-2020 Magento, Inc. (https://www.magento.com)
+ * @copyright  Copyright (c) 2024 The OpenMage Contributors (https://www.openmage.org)
+ * @license    https://opensource.org/licenses/afl-3.0.php Academic Free License (AFL 3.0)
+ */
+-->
+<widgets>
+    <newsletter_subscribe type="newsletter/widget_subscribe" translate="name description" module="newsletter">
+        <name>Newsletter Subscribe Form</name>
```

Comment:
I'm not a native speaker, it's usually "Newsletter Subscription Form". Or it's ok as is?